styles.css

```diff
@@ -416,3 +441,849 @@ textarea {
+
+  .deskcontact {
+    display: flex;
+    gap: 1%;
+    justify-content: flex-end;
+  }
+
+  .deskcontact :nth-child(1) {
+    margin-right: auto;
+  }
+
+  .deskcontact input {
+    min-width: 45%;
+  }
+
+  form button {
+    margin-left: 0;
+    padding: 1%;
+    width: 25%;
+  }
+
+  .hra {
+    margin-top: 20%;
+  }
+
+  .languagescenter {
+    display: flex;
+    justify-content: space-evenly;
+    text-align: left;
+  }
+
+  header a {
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .frontpage {
+    margin-bottom: 10%;
+  }
+
+  main {
+    background-color: #1c1a19;
+    background-image: url(./images/desky/group.svg);
+    background-size: 45%;
+    background-repeat: no-repeat;
+    background-position: top 0% right;
+    padding-top: 5%;
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: baseline;
+    justify-content: flex-end;
+    padding-bottom: 2%;
+  }
+
+  .bars {
+    display: none;
+  }
+
+  .logo {
+    margin-left: 10%;
+  }
+
+  header {
+    background-color: #3c3a39;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    padding-bottom: 1%;
+    padding-top: 1%;
+  }
+
+  .frontpage ul {
+    flex-direction: column;
+  }
+
+  main section {
+    width: 50%;
+  }
+
+  .recent {
+    color: #1c1a19;
+    line-height: 52px;
+    border-bottom: none;
+    text-align: left;
+    padding: 0;
+    margin: 0;
+  }
+
+  .line {
+    display: flex;
+    gap: 2%;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 5% 5%;
+  }
+
+  .line img {
+    max-width: 50%;
+  }
+
+  .card img {
+    width: 100%;
+  }
+
+  .projimage {
+    display: inline;
+    float: left;
+    width: 90%;
+  }
+
+  .card button {
+    padding: 1%;
+    background-color: #ff6b00;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+    border: none;
+    width: 40%;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    gap: 2%;
+    margin: 0;
+    padding: 0;
+    width: 80%;
+  }
+
+  .text {
+    max-width: 40%;
+    padding-right: 0;
+  }
+
+  .cardheader {
+    margin-top: 0;
+  }
+
+  .card li {
+    padding: 2%;
+  }
+
+  .grids {
+    width: 80%;
+    display: flex;
+    gap: 4%;
+    flex-wrap: wrap;
+    margin-top: 5%;
+    padding-bottom: 0;
+  }
+
+  .card2 {
+    background-image: url(./images/projects/project\ background.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 30%;
+    color: #fff;
+    padding-top: 0;
+    margin-top: 0;
+    margin-left: 0;
+    margin-right: 0;
+    margin-bottom: 0;
+    padding-bottom: -90%;
+  }
+
+  .card2 p {
+    font-size: 17px;
+    margin-top: 0;
+    width: 100%;
+  }
+
+  .card2 h2 {
+    font-size: 32px;
+    margin-bottom: 0;
+  }
+
+  .card2 li {
+    font-size: 10%;
+    padding: 3%;
+  }
+
+  .card2 ul {
```

Comment:
- [ ] To match the Figma design as it is required for the project, kindly make sure all your cards  design stick as much as possible to the one form the Figma template. For each card , the `See Project` button has to appear when you hove the card and it should have the border as well.
### Template Design
![Screenshot from 2023-06-08 19-09-23](https://github.com/mj-e-boc/M-J-Portfolio/assets/83042742/273fc0f9-e129-4ab8-980f-e0d26351e021)

### Your Design
![Screenshot from 2023-06-08 18-55-33](https://github.com/mj-e-boc/M-J-Portfolio/assets/83042742/302c0af0-0d2b-4c6e-8d01-c3fd633376e0)

- [ ]  Kindly ensure that your Button interactions such as `hover`, `active` (when it's pressed), `disabled` and `enable` (default state) are styled as shown in the Figma template for the project requirements. You should apply this approach to both DESKTOP and MOBILE
![Screenshot from 2023-06-08 19-19-57](https://github.com/mj-e-boc/M-J-Portfolio/assets/83042742/9a879018-07db-4b9f-99b8-d1086003db04)